core/pipeline/cms/cms.proto

```diff
@@ -120,30 +120,30 @@ message CmsListNsResponse {
 message CmsNsConfigsUpdateRequest {
   string ns = 1;
   string pipeline_source = 2 [(validator.field) = {string_not_empty: true}];
-  map<string, PipelineCmsConfigValue> kvs = 3;
+  map<string, PipelineCmsConfigValue> KVs = 3 [json_name="kvs"];
 }
 message CmsNsConfigsUpdateV1Request {
   string ns = 1;
-  map<string, string> kvs = 2;
+  map<string, string> KVs = 2 [json_name="kvs"];
 }
 message CmsNsConfigsUpdateResponse {
 }
 
 message CmsNsConfigsDeleteRequest {
   string ns = 1;
-  string pipeline_source = 2 [(validator.field) = {string_not_empty: true}];
-  bool delete_ns = 3 [json_name="deleteNS"];
-  bool delete_force = 4;
-  repeated string delete_keys = 5;
+  string pipelineSource = 2 [(validator.field) = {string_not_empty: true}];
+  bool deleteNs = 3 [json_name="deleteNS"];
```

Comment:
why not change json name

Reply:
json_name for backward-compatible.
use `Ns` because all the other places use this format.

Reply:
so you should comment it in code

Reply:
Added.